src/pybind/mgr/cephadm/cert_mgr.py

```diff
@@ -0,0 +1,46 @@
+
+from cephadm.ssl_cert_utils import SSLCerts, SSLConfigException
+from threading import Lock
+from typing import TYPE_CHECKING, Tuple, Union, List
+
+if TYPE_CHECKING:
+    from cephadm.module import CephadmOrchestrator
+
+
+class CertMgr:
+
+    CEPHADM_ROOT_CA_CERT = 'cephadm_root_ca_cert'
+    CEPHADM_ROOT_CA_KEY = 'cephadm_root_ca_key'
+
+    def __init__(self, mgr: "CephadmOrchestrator", ip: str) -> None:
+        self.lock = Lock()
+        self.initialized = False
+        with self.lock:
```

Comment:
is this locking mechanism necessary? As far as I can tell, we only initialize this object in one place at module startup (which runs once then never again unless the module is restarted). Then the other uses of the `self.initialized` variable seem to be in a read only fashion.

Reply:
yeah, this code has changed a little bit during the dev of the feature and your are right that the lock is not necessary anymore. I removed the lock and the `initialized` variable as well because whenever the certmgr is called module should be already initialized. BTW: this call can comes from other modules (such as prometheus) when asking cephadm to create certificates for them.